src/index.less

```
@@ -1,476 +1,338 @@
+      color: @sidebar-actions-button-font-color;
+      font-weight: @sidebar-actions-button-font-weight;
+      font-size: @sidebar-actions-button-font-size;
       text-align: left;
-      font-size: @SIDEBAR_MENU_ACTIONS_MENU_FONT_SIZE;
 
-      &.menu-open {
-        background-color: #fff;
-        color: @SIDEBAR_MENU_ACTIONS_MENU_OPEN_COLOR;
-        outline-color: @SIDEBAR_MENU_ACTIONS_MENU_OPEN_COLOR;
+      &:focus {
+        outline-width: 1px;
+        outline-style: dotted;
+        outline-color: initial;
+        outline-offset: 2px;
       }
 
       &:hover,
       &:active,
       &:focus {
-        &:not(.menu-open) {
-          background-color: darken(@SIDEBAR_MENU_ACTIONS_MENU_BG, 12%);
-          border-color: darken(@SIDEBAR_MENU_ACTIONS_MENU_BG, 12%);
-        }
+        background-color: @sidebar-actions-button-background-color_hover;
+        border-color: @sidebar-actions-button-border-color_hover;
+        color: @sidebar-actions-button-font-color_hover;
       }
 
-      > .cart-icon,
-      > .button-text {
-        padding-right: 8px;
+      &.menu-open {
+        background-color: @sidebar-actions-button-background-color_active;
+        border-color: @sidebar-actions-button-border-color_active;
+        color: @sidebar-actions-button-font-color_active;
       }
 
-      > .arrow-icon {
-        float: right;
-        font-size: 8px;
+      .button-text {
+        padding: @sidebar-actions-button-text-padding;
+        flex: 1;
+      }
 
-        > i {
-          padding-top: 6px;
-        }
+      .arrow-icon {
+        font-size: @sidebar-actions-button-icon-size;
       }
     }
-  }
-
-  // shared styles for all level
-  .sidebar-menu-list,
-  .sub-menu {
-    padding: 0;
-    margin: 0;
-    list-style-type: none;
 
-    li {
-      color: @SIDEBAR_MENU_COLOR;
-
-      .menu-item {
-        color: @SIDEBAR_MENU_COLOR;
-        background-color: transparent;
-        border: 0;
-        text-decoration: none;
-        display: block;
-        white-space: nowrap;
-        text-overflow: ellipsis;
-        overflow: hidden;
-        line-height: @SIDEBAR_MENU_ITEM_HEIGHT;
-        max-height: @SIDEBAR_MENU_ITEM_HEIGHT;
-
-        .item-arrow,
-        .item-loading,
-        .item-icon {
-          width: 20px;
-          text-align: center;
-          display: inline-block;
-          font-size: 8px;
-        }
+    // Actions - Menu
+    .responsive-popover.order-actions-menu-popover {
+      margin: @sidebar-actions-menu-margin;
+      border: solid @sidebar-actions-menu-border-width @sidebar-actions-menu-border-color;
 
-        .item-loading > .oui-spinner {
-          vertical-align: middle;
-        }
+      .arrow {
+        display: none;
+      }
 
-        .item-arrow > i {
-          margin-bottom: 4px;
+      .actions-list .actions-menu-item-link {
+        .oui-icon {
+          display: inline;
+          color: inherit;
         }
 
+        // Force styles to avoid stupidly long selectors
```

```suggestion
        // Force styles to avoid unnecessary long selectors
```

The downside is that we are disabling the rule `no-descending-specificity` from the stylelint configuration file.

As the whole component style deserve to be refactored later on, I'm agree with theses selectors.

---

I disable `no-descending-specificity` because it returns false positive